### DISCUSSION TITLE
Where is "isInEditMode"

### DISCUSSION THREAD

**Discussion by rozrabiak** [forntoh/LcdMenu#183] [Q&A] [answered]
Hi, after update to 4.0.0 I can't find:
"isInEditMode" and "edit"

And where is:
menu.lcd->backlight();
menu.lcd->noBacklight();

:-(
What I must change in code?

**Comment by forntoh** [comment]
https://github.com/forntoh/LcdMenu/discussions/181#discussioncomment-10435070

**Reply by rozrabiak** [reply]
in above link and in "Migration guide" and in other documentation - there is no information about "backlight", "noBacklight" and how to check if is in edit mode or enter edit mode in 4.0.0 version.

Sorry, but You answer is not useful.
Below, all functions for menu.lcd.

![image](https://github.com/user-attachments/assets/42db7115-e4d6-400c-9ca8-70ce20f81da9)

**Reply by forntoh** [reply, marked as answer]
<img width="791" alt="Screenshot 2024-09-01 at 11 31 06" src="https://github.com/user-attachments/assets/44b98445-3446-42ec-bfa5-cd1ebd3b5ce9">

**Reply by forntoh** [reply]
<img width="1160" alt="image" src="https://github.com/user-attachments/assets/a1e52ec6-4320-4059-a49c-3facb77afcd1">
All functions of LiquidCrystal/LiquidCrystalI2C can be accessed from the lcd instance in lcdAdapter (that's what the info block explains), so similar to `setBacklight`, `backlight` and `noBacklight` can be accessed.
